amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/disposition/DispositionServiceImpl.java

```diff
@@ -1191,7 +1180,7 @@ public void cutoffDisposableItem(final NodeRef nodeRef)
         if (FilePlanComponentKind.RECORD_FOLDER.equals(filePlanService.getFilePlanComponentKind(nodeRef)) ||
             FilePlanComponentKind.RECORD.equals(filePlanService.getFilePlanComponentKind(nodeRef)))
         {
-            if (!isDisposableItemCutoff(nodeRef) && !isFrozenOrHasFrozenChildren(nodeRef))
+            if (!isDisposableItemCutoff(nodeRef))
```

Comment:
I don't understand this change. The ticket talks about preventing actions for frozen records, but this change seems to do the opposite.

Reply:
This check is moved from this service to DispositionLifeCycleJobExecutor.java

Reply:
This ticket talks about preventing action for frozen records, this we have handled in the another class DispositionLifeCycleJobExecutor.java for the better code refactoring. Also we covered basic testing for this functionality.

Reply:
Ok sounds good. We should ensure this gets some exploratory testing in case there are other routes to get to this method.